presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -230,6 +230,7 @@
     public static final String KEY_BASED_SAMPLING_FUNCTION = "key_based_sampling_function";
     public static final String HASH_BASED_DISTINCT_LIMIT_ENABLED = "hash_based_distinct_limit_enabled";
     public static final String HASH_BASED_DISTINCT_LIMIT_THRESHOLD = "hash_based_distinct_limit_threshold";
+    public static final String ROUND_ROBIN_SHUFFLE_BEFORE_PARTIAL_AGG = "round_robin_shuffle_before_partial_agg";
```

Comment:
I would call this `shuffle_before_partial_distinct_limit` since it's specific to distinct limit, not all partial aggs.